controller/service.go

```diff
@@ -63,31 +69,56 @@ func (s *Service) CreateBirthday(ctx context.Context, req *pb.CreateBirthdayRequ
 	return convertedBirthday, nil
 }
 
+//GetBirthday is getting a birthday object
 func (s *Service) GetBirthday(ctx context.Context, req *pb.GetBirthdayRequest) (*pb.BirthdayObject, error) {
-	birthdayCollection := mongoConnect.CNX.Database("birthdayDB").Collection("birthdays")
 
-	birthday := birthdayCollection.FindOne(ctx, bson.M{"personalNumber": req.PersonalNumber})
-	BirthdayModel := &BirthdayStruct{}
+	birthday := s.BirthdayCollection.FindOne(ctx, bson.M{"personalNumber": req.PersonalNumber})
+	BirthdayModel := &BirthdayModel{}
 	birthday.Decode(BirthdayModel)
 	convertedBirthday := BirthdayModel.asBirthdayObject()
 
 	return convertedBirthday, nil
 }
 
+//GetAllBirthday is creating all birthday objects
+func (s *Service) GetAllBirthday(ctx context.Context, req *pb.GetAllBirthdayRequest) (*pb.GetAllBirthdayResponse, error) {
+
+	cursor, err := s.BirthdayCollection.Find(ctx, bson.M{})
+	if err != nil {
+		log.Fatal(err)
+	}
+	var mongoBirthdays []BirthdayModel
+	if err = cursor.All(ctx, &mongoBirthdays); err != nil {
+		log.Fatal(err)
+	}
+	birthdaysArray := make([]*pb.BirthdayObject, 0, len(mongoBirthdays))
+
+	for index := 0; index < len(mongoBirthdays); index++ {
```

Comment:
What does this for loop do? Add documentation.

Reply:
done